pootle/apps/reports/views.py

```diff
@@ -204,6 +204,12 @@ def get_detailed_report_context(user, month):
                 else:
                     totals['reviewed'][score.review_rate] = {'words': reviewed}
 
+            suggested = score.get_suggested_words()
+            if suggested is not None:
+                score.action = _('Suggestion')
```

Comment:
I see this is not limited to this PR but to the whole view in general: I'd highly advise against mutating `score` — it can have unexpected side-effects. Also you start relying on some implicit contract set somewhere in the view (remember explicit > implicit).

Since in the end this `score` is being used to populate some context that will be fed to the template, I'd rather create a dictionary which contains the values needed by the template.